A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_WHEEL.js

```diff
@@ -358,6 +302,18 @@ var A320_Neo_LowerECAM_WHEEL;
             }
         }
 
+        updateBrakeTemp(_deltaTime) {
+            this.currentBrake1Temp = Math.round(SimVar.GetSimVarValue("L:A32NX_BRAKE_TEMPERATURE_1", "celsius"));
+            this.currentBrake2Temp = Math.round(SimVar.GetSimVarValue("L:A32NX_BRAKE_TEMPERATURE_2", "celsius"));
+            this.currentBrake3Temp = Math.round(SimVar.GetSimVarValue("L:A32NX_BRAKE_TEMPERATURE_3", "celsius"));
+            this.currentBrake4Temp = Math.round(SimVar.GetSimVarValue("L:A32NX_BRAKE_TEMPERATURE_4", "celsius"));
```

Comment:
I don't think those are defined, since you deleted the definiton above. Should be all const.

Reply:
ok, can't be const, since this.currentBrake*Temp is use elsewhere. Needs to be defined